nuget/lib/dependabot/nuget/file_parser/project_file_parser.rb

```diff
@@ -408,10 +408,6 @@ def package_versions
                 version = dependency_version(package_node, file)
                 next unless name && version
 
-                version = Version.new(version)
-                existing_version = package_versions[name]
-                next if existing_version && existing_version > version
-
```

Comment:
Is the thinking that last version wins here?

Reply:
nevermind. I see the comment in the description.

Reply:
Correct, similar to multiple property definitions, last wins.